codersdk/deployment.go

```diff
@@ -1287,13 +1287,13 @@ func (c *DeploymentValues) Options() serpent.OptionSet {
 			Annotations: serpent.Annotations{}.Mark(annotationFormatDuration, "true"),
 		},
 		{
-			Name:        "Job Hang Detector Interval",
-			Description: "Interval to poll for hung jobs and automatically terminate them.",
+			Name:        "Job Reaper Detect Interval",
+			Description: "Interval to poll for hung and pending jobs and automatically terminate them.",
 			Flag:        "job-hang-detector-interval",
 			Env:         "CODER_JOB_HANG_DETECTOR_INTERVAL",
 			Hidden:      true,
 			Default:     time.Minute.String(),
-			Value:       &c.JobHangDetectorInterval,
+			Value:       &c.JobReaperDetectorInterval,
```

Comment:
Follow-up suggestion, non-blocking: there is now a disconnect between our internal naming ("job reaper") and the external flags ("hang detector"). We may wish to deprecate the old env in future to avoid confusion.